.github/workflows/ci_cron_daily.yml

```diff
@@ -57,19 +58,20 @@ jobs:
 
         repo-token: ${{ secrets.GITHUB_TOKEN }}
         operations-per-run: 50
-        ascending: true
+        ascending: false
 
         days-before-issue-stale: -1
         days-before-issue-close: 7
         remove-issue-stale-when-updated: false
         close-issue-message: 'I am going to close this issue because it was marked with **Close?**, but if you feel that this issue should stay open, then feel free to re-open, remove **Close?** and **closed-by-bot** labels, and apply **keep-open** label.'
+        stale-issue-label: 'Close?'
         close-issue-label: 'closed-by-bot'
         exempt-issue-labels: 'keep-open'
         any-of-issue-labels: 'Close?'
 
         days-before-pr-stale: 150
         days-before-pr-close: 30
-        remove-pr-stale-when-updated: true
+        remove-pr-stale-when-updated: false
```

Comment:
This set to `true` basically causes stale PR not to be marked stale ever, due to how we do things here. Until upstream is fixed, we cannot use this option. See https://github.com/actions/stale/issues/435#issuecomment-845313819 based on inspection of how #9856 was handled.